### ISSUE TITLE
MQTT payloads and config example

### ISSUE THREAD

**Issue by Grafton1046** [AntoineGS/teams-status-rs#24]
If possible can the the payloads for the various sensors be in all caps, home assistant expects them in all caps to use the default payload detection.
I would prefer to have the API sensors tied into one device on the HA side, so the icons could change, not just the colors, and have everything grouped together. Digging into that on my own seemed like a complete integration would have to be made, so that may just be a pipe dream

Attached is a sample config for mqtt binary sensors that tie everything into one device in home assistant.
![mqtt device](https://github.com/AntoineGS/teams-status-rs/assets/85599782/53a118c7-ee96-4381-b9a3-9a6a43e77e5a)
https://i.imgur.com/ZF5rJc3.png

**Comment by AntoineGS** [comment]
Yeah I really like the idea of having them under one device, I will definitely do this for MQTT as it seems straightforward.
I'll look into the HA API too but unless I find something you did not I will likely not implement it.
Thanks for the feedback!
